README.md

```diff
@@ -146,24 +146,61 @@ If `NODE_ENV` is not set when ez-build is invoked in production mode, it will be
 
 *Note: the `NODE_ENV=production ez-build` syntax may not work on all systems. For compatibility across operating systems, you may want to consider using something like [cross-env](https://www.npmjs.com/package/cross-env).*
 
+### `--target-browsers <spec|false>`
+
+Define which browsers that ez-build should do its best to target when producing its output. What this means is that only those features that actually need compiling will be compiled, and the rest will be left alone. The `spec` string can be any query supported by [browserslist](https://github.com/ai/browserslist). This option defaults to `"last 3 versions"` which is a rather broad range, intended to capture most reasonably modern browsers.
+
+**Please note:** because of the rather freeform nature of target queries, it's very likely that you will have to quote the query, otherwise it won't be passed correctly. For instance, this will not be recognized properly:
+
+```bash
+$ ez-build --target-browsers last 3 versions
```

Comment:
Don't put bad examples at the top, as people will just copy paste the first one with the thought of "tl;dr".